index.html

```diff
@@ -6,7 +6,133 @@
+<ul class="advantages-list">
+  <li class="advantages-item">
+    <h2>Седона — небольшой городок в Аризоне, заслуживающий большего!</h2>
+    <span>Рассмотрим 5 причин, по которым Седона круче, чем гранд каньон!</span>
+    <h3 class="advantages-title"> Настоящий городок</h3>
+    <p class="advantages-description">— №1 — Седона — не аттракцион для туристов, там течёт
+      своя жизнь</p>
+      <img src="" alt="городок">
+      <ul class="features-list">
+        <li class="features-iteam">
+        <img class="icon" src="" alt="Жильё">
+        <h3 class="features-title">Жильё</h3>
+        <p class="features-description">Рекомендуем пожить<br> в настоящем мотеле,<br> всё как в кино!</p>
+        </li>
+        <li class="features-iteam">
+          <img class="icon" src="" alt="Еда">
+          <h3 class="features-title">Еда</h3>
+          <p class="features-description">Всегда заказывайте<br> топовый фирменный бургер,<br> вы не разочаруетесь!</p>
+          </li>
+          <li class="features-iteam">
+            <img class="icon" src="" alt="Сувениры">
+            <h3 class="features-title">Сувениры</h3>
+            <p class="features-description">Не только китайского,<br> но и настоящего местного<br> производства!</p>
+            </li>
+      </ul>
+  </li>
+  <li class="advantages-item">
+    <img src="" alt="мост">
+    <h3 class="advantages-title"> Там есть мост дьявола</h3>
+    <p class="advantages-description">— №2 — <br>Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+  </li>
+  <li class="advantages-item">
+    <h3 class="advantages-title"> Небольшая площадь</h3>
+    <p class="advantages-description">— №3 —<br>Все достопримечательности<br>находятся очень близко</p>
+  </li>
+  <li class="advantages-item">
+    <h3 class="advantages-title"> Красивая дорога</h3>
+    <p class="advantages-description"> — №4 —<br>Ехать в Седону из Лас-Вегаса<br> совсем не скучно!</p>
+  </li>
+  <li class="advantages-item">
+    <h3 class="advantages-title"> Мало туристов</h3>
+    <p class="advantages-description"> — №5 —<br>Большинство едет в Гранд<br> Каньон и толпится там</p>
+  </li>
+</ul>
+<section class="Search">
+  <h2 class="search-tiile">Заинтересовались?</h2>
+  <p>Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+<a class="search-link" href="">Поиск гостиницы в Седоне</a>
+</section>
+<section class="mailing">
+  <h2 class="mailing-tiile">Подпишитесь на рассылку</h2>
+  <p> Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
+  <form class="mailing-form" action="" method="post">
+  <label class="visually-hidden" for="mailing-email">Email</label>
+  <input class="field" placeholder="email@exemple.com" type="email" name="mailing-email" id="mailing-email" required>
+  <button class="button mailing-button" type="submit">
+    <span class="visually-hidden">Подписаться</span>
+  </button>
+</form>
+</section>
+</main>
+<footer class="page-footer"
```

Comment:
Закрывающую скобку потеряла
```suggestion
<footer class="page-footer">
```

Reply:
Исправила.